views/js/pciManager/pciManager.js

```diff
@@ -290,42 +323,42 @@ define([
                         }
                     });
 
-                    function verify(file, cb){
+                    function verify(file, cb) {
```

Comment:
It could be better not to make so many whitespace changes in the parts of the file you don't need to change. People looking at the code blame might assume all this is your work!